copier/copier.go

```diff
@@ -1785,3 +1831,22 @@ func copierHandlerMkdir(req request, idMappings *idtools.IDMappings) (*response,
 
 	return &response{Error: "", Mkdir: mkdirResponse{}}, nil, nil
 }
+
+func copierHandlerRemove(req request) *response {
+	errorResponse := func(fmtspec string, args ...interface{}) *response {
+		return &response{Error: fmt.Sprintf(fmtspec, args...), Remove: removeResponse{}}
+	}
+	resolvedTarget, err := resolvePath(req.Root, req.Directory, false, nil)
+	if err != nil {
+		return errorResponse("copier: remove: %v", err)
+	}
+	if req.RemoveOptions.All {
+		err = os.RemoveAll(resolvedTarget)
+	} else {
+		err = os.Remove(resolvedTarget)
+	}
+	if err != nil {
+		return errorResponse("copier: remove %q: %v", req.Directory, err)
```

Comment:
Still a bit of a stutter, but I guess req.Directory != resolvedTarget
/lgtm
/hold